app/src/main/java/com/pose/move/feature/auth/register/RegisterEvent.kt

```diff
@@ -9,4 +9,6 @@ sealed class RegisterEvent {
     class PasswordChanged(val password: String) : RegisterEvent()
 
     object Register : RegisterEvent()
+
+    object ClearError : RegisterEvent()
```

Comment:
You could convert the `RegisterEvent` class into an `sealed interface` instead of class, since you don't take any values in the constructor.